docs/posts/demo-data.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: demo-data
+description: Demo datasets
+---
+
+# Example datasets
+**To download and use these datasets, run:**  
+`curl -O [URL]`
+`unzip [filename.zip]`
+`cellxgene launch [filename.h5ad] --open`
+
+### Peripheral blood mononuclear cells
+Healthy human PBMCs (10X).
+
+- Source: [10X genomics](https://support.10xgenomics.com/single-cell-gene-expression/datasets/1.1.0/pbmc3k)
+- Cells: 2,638
+- File size: 19MB
+- [Raw data](http://cf.10xgenomics.com/samples/cell-exp/1.1.0/pbmc3k/pbmc3k_filtered_gene_bc_matrices.tar.gz)
+- [Processing](https://github.com/chanzuckerberg/cellxgene-vignettes/blob/master/dataset-processing/pbmc3k-processing.ipynb)
+- Download: `curl -O https://cellxgene-example-data.czi.technology/pbmc3k.h5ad.zip`
+
```

Comment:
@bkmartinjr @liaprins-czi -- The suggestion to just `cellxgene launch [url]` for these is a great one! I hesitated because they're hosted on AWS, meaning that would require an additional install step to launch directly. Can I get a vote on this one? I'll update asap either way.

Reply:
if they have a publicly accessible http URL (ie, you can fetch with curl), then you don't need an extra install. The extra install is only for the s3: URL support.

Did you experiment and find that it failed or something like that?

Reply:
Ah, cool. I'd still have to unzip them; that would be fine for pbmc3k and tabula muris, but I'd have to remove tabula muris senis (the uncompressed download is quite wieldy). 

For now, I'll leave them as-is, but we can revisit later.

Reply:
Yes, needs to be unzipped, but the files are already pretty small, so the zipping isn't a huge win.   

I'm fine deferring this!